We are not interested in these devel-only checks because we do not have some
required projects packaged.

--- jaraco_mongodb-12.4.0/tox.ini.orig
+++ jaraco_mongodb-12.4.0/tox.ini
@@ -10,10 +10,8 @@
 usedevelop = True
 extras =
 	test
-	check
 	cover
 	enabler
-	type
 
 [testenv:diffcov]
 description = run tests and check that diff from main is covered
